.github/workflows/publish.yml

```diff
@@ -7,6 +7,7 @@ on:
 
 permissions:
   contents: write
+  id-token: write
```

Comment:
What does it provide?

Reply:
this also mistake, i forgot to remove it